types/owlcarousel/index.d.ts

```diff
@@ -29,7 +29,7 @@ interface IOwlCarouselOptions {
     scrollPerPage?: boolean;
     pagination?: boolean;
     paginationNumbers?: boolean;
-    responsive?: boolean;
+    responsive?: boolean | any;
```

Comment:
`boolean | any` is just the same as `any`. Did you mean `boolean | object` ? Can this accept `string` ?

Reply:
My answer via email is not attached to your question https://github.com/DefinitelyTyped/DefinitelyTyped/pull/22794#issuecomment-357007850

Reply:
What do you mean by "it can be an object"? How does the library use the value? If it is just checked for truthiness we should leave the type declaration as `boolean`.

Reply:
After reading the documentation (useless) and the code, the closest thing to a correct type is `false | object`